website/docs/r/rde_type.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_rde_type"
+sidebar_current: "docs-vcd-resource-rde-type"
+description: |-
+   Provides the capability of creating, updating, and deleting Runtime Defined Entity types in VMware Cloud Director.
+---
+
+# vcd\_rde\_type
+
+Provides the capability of creating, updating, and deleting Runtime Defined Entity types in VMware Cloud Director.
+Requires system administrator privileges.
+
+Supported in provider *v3.9+*
+
```

Comment:
Would we want to add a note about operations on RDE type being run in serial due to API? Or is that too much for the user for this use case? How long does it take to create these types in practice?

CC: @dataclouder @Didainius @adezxc

Reply:
I don't think it's needed, even they run sequentially they're created almost immediately. It is not like VMs, for example, which require some time to be created.

Reply:
I don't think it's needed, even they run sequentially they're created almost immediately. It is not like VMs, for example, which require some time to be created.

Reply:
I was going to say we don't need that, but just thinking if there can ever be such situation that RDE types are created from different Terraform instances (then locks will not work).

Not sure if we're too cautious here.

Reply:
Too small probability for now, IMHO.